source/includes/stackpath/_firewallrules.md

```diff
@@ -0,0 +1,46 @@
+## Firewall Rules
+
+### Create a firewall rule
+
+Restrict access to a site using allow and block rules.
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Required Query Params | &nbsp;
+---- | -----------
+`siteId`<br/>*UUID* | The ID of the site for which to create the firewall rule. This parameter is required.
+
+Required Body Attribute | &nbsp;
+------------------------| -----------
+`action`<br/>*string* | Either ALLOW or BLOCK.
+`name`<br/>*string* | The name of the rule.
+`ipStart`<br/>*string* | There start ip adress for the rule. When no `ipEnd` attribute is provided, the rule only applies for the ip provided in `ipStart`.
```

Comment:
`The` instead of `There` here and on line 21?

Reply:
Good catch @aaronalmeida. Fixed in 99385d92af6bad0d88e4a864f510f0e54759ea44